### DISCUSSION TITLE
SFDX not working on Mac

### DISCUSSION THREAD

**Discussion by MrN1gma** [microsoft/terminal#15352] [General]
I have been trying to use the sfdx commands on Mac with all the necessary extensions installed, however I still cannot get the commands to pop up and keep getting the error message  zsh: command not found: sfdx as well as zsh: command not found: list
I have tried to contact Microsoft about this but they shifted the blame to Github so I would appreciate if anyone could help resolve this issue

**Comment by DHowett** [comment]
Hi,
Thanks for starting a discussion! Unfortunately, _this team doesn't know what SFDX is_ and, furthermore, we don't really work on any Mac software over here. We're the "Windows Terminal" team, who works on an application named "Windows Terminal". This repository is specific to "Windows Terminal".

**Reply by MrN1gma** [reply]
Where would be the best place to discuss this issue then.

**Reply by zadjii** [reply]
[where to file sfdx issues](https://www.bing.com/search?q=where+to+file+sfdx+issues&qs=n&form=QBRE&sp=-1&lq=0&pq=where+to+file+sfdx+issues&sc=8-25&sk=&cvid=E1D1623CAA0B4A418D13B4698AF13C9F&ghsh=0&ghacc=0&ghpl=) perhaps?

**Reply by DHowett** [reply]
Given that I do not know what SFDX is (and a cursory online search did not turn up anything helpful,) I don't know where to redirect you. Sorry.

**Reply by MrN1gma** [reply]
what about to do with Microsoft help seeing as Visual Source Code is made by Microsoft

**Reply by DHowett** [reply]
I recommend using the [VSCode repository](https://github.com/microsoft/vscode).

Thanks! Good luck!
